compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -423,6 +423,7 @@ MemRefConversionPattern<OpTy>::getVectorizedMemRefType(
       if (index == strides.size() - 1 || stride == ShapedType::kDynamic) {
         continue;
       }
+      strides[index] = stride / ratio;
```

Comment:
good example of why this type of manipulations are tricky and should be centralized in a followup :)